addons/sale_project/models/project.py

```diff
@@ -769,7 +785,9 @@ def action_open_project_vendor_bills(self):
             'views': [[False, 'tree'], [False, 'form'], [False, 'kanban']],
             'domain': [('id', 'in', vendor_bill_ids)],
             'context': {
-                'create': False,
+                'default_move_type': 'in_invoice',
+                'default_partner_id': self.partner_id.id,
```

Comment:
We should remove that since the customer set on the project is not really the supplier. 🙂